NuGet.Config

```diff
@@ -27,6 +27,7 @@
     <add key="myget.org roslyn-master-nightly" value="https://dotnet.myget.org/F/roslyn-master-nightly/api/v3/index.json" />
     <add key="myget.org devcore" value="https://www.myget.org/F/vs-devcore/api/v3/index.json" />
     <add key="myget.org roslyn-tools" value = "https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
+    <add key="dotnet.myget.org msbuild" value="https://dotnet.myget.org/F/msbuild/api/v3/index.json" />
```

Comment:
That's fine.  But either we care about this and hence change everything or we don't and keep the existing pattern. 

---
In reply to: [75893767](https://github.com/dotnet/roslyn/pull/13288#discussion_r75893767) [](ancestors = 75893767)